configs/projects/_shared-pe-bolt-server.rb

```diff
@@ -105,6 +105,7 @@
 proj.component('rubygem-log4r')
 proj.component('rubygem-logging')
 proj.component('rubygem-minitar')
+proj.component('rubygem-molinillo')
```

Comment:
Can you file a RE ticket with a new library request: 
![image](https://user-images.githubusercontent.com/15354214/76095801-c04a5580-5f79-11ea-854c-b41263cd5b1a.png)

Reply:
The external gem i'm sure is fine (MIT licence) but I think all "new" gems shipped with PE will need to go through this process (it is very quick).

Reply:
Good to know. 
molinillo: https://tickets.puppetlabs.com/browse/RE-13299
puppetfile-resolver: https://tickets.puppetlabs.com/browse/RE-13300